packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: url_launcher
 description: Flutter plugin for launching a URL on Android and iOS. Supports
   web, phone, SMS, and email schemes.
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher
-version: 5.4.4
+version: 5.4.5
```

Comment:
nit: Should this be a patch or a minor patch bump? If you're intending for this to be a patch, all the ones that are under 1.0.0 need to have a +1 appended. If it's intended to be minor, all the ones above 1.0.0 should have the middle digit incremented. Right now it looks like the last digit is always being incremented, but with how pub handles dependencies that ends up being inconsistent.

I think it makes the most sense for this to be a patch upgrade because it's bumping a dependency (semver says that's a patch upgrade on its own) and fixing an internal issue. But I can see it going the other way.

Reply:
My thoughts exactly when looking through this.  Should upgrading flutter dependency increment the patch or minor number?  I don't know the answer but minor seem more right to me.

Reply:
Yeah, I don't think there's a single right answer. Here's the semver spec on it: https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api. That suggests patch in most cases. Since this is the Flutter SDK itself though I can see the argument for it being a minor bump, because in a roundabout way it _is_ part of the public API of the plugin, since everyone needs to have at least that Flutter SDK version to get the plugin version.

Reply:
I thought about bumping the minor for the same reason you described. If there's a strong reason for bumping the patch, I can definitely do that.